src/pages/stop-details/stop-details.ts

```diff
@@ -141,15 +142,31 @@ export class StopDetailsPage {
   }
 
   MapbuttonClick() {
-    var stopInfo = {'stopName': this.stopName,
-                    'location': this.stopLocation};
-                    
-    let mapModal = this.modalCtrl.create(GoogleMapPage,stopInfo);
+    var stopInfo = {
+      'stopName': this.stopName,
+      'location': this.stopLocation
+    };
+
+
+    let mapModal = this.modalCtrl.create(GoogleMapPage, stopInfo);
     mapModal.present();
   }
 
-  buttonClick() {
+  addFav(stopName) {
+    var tempData = ',' + stopName;
```

Comment:
not a huge fan of this data structure, **but it's fine for now.** 

ideally should have used something like this if we could store an object as the value of a key
'favstops': {
'weber arch': true,
'tech institute': true,
...
}